src/SFA.DAS.AANHub.Api/Controllers/ApprenticesController.cs

```diff
@@ -37,7 +38,30 @@ public async Task<IActionResult> CreateApprentice([FromHeader(Name = Constants.P
             command.RequestedByMemberId = userId;
 
             var response = await _mediator.Send(command);
-            return new CreatedAtActionResult(nameof(CreateApprentice), "Apprentices", new { id = response.MemberId }, response);
+            return new CreatedAtActionResult(nameof(ApprenticesController.GetApprentice), "Apprentices", new { apprenticeid = request.ApprenticeId }, response);
```

Comment:
I think this should still be `ApprenticesController.CreateApprentice`?

Additionally, I think you'll need to update this workflow with the updated Validation system as per #30 . Check the employer work in that PR, and let me know if you have questions.